### PR TITLE
Changed option names according to the option style guide

### PR DESCRIPTION
**Pull request by LSLindeloo**
fixes https://github.com/jasp-stats/INTERNAL-jasp/issues/2044
fixes https://github.com/jasp-stats/INTERNAL-jasp/issues/2045
fixes https://github.com/jasp-stats/INTERNAL-jasp/issues/2046
fixes https://github.com/jasp-stats/INTERNAL-jasp/issues/2047

### CODE REVIEW

**Comment by LSLindeloo** [comment]
Somehow, the unit tests fail on macOS because the tables get filled with zeros, do you know how to solve this?

File: inst/qml/SEM.qml [hunk 1]
```diff
@@ -143,33 +143,33 @@ Form
...
+					{ label: qsTr("Factor loadings")	, value: "factorLoading"	},
+					{ label: qsTr("Factor variance")	, value: "factorVariance"			},
+					{ label: qsTr("Effects coding")		, value: "effectCoding"	},
 					{ label: qsTr("None")				, value: "none"				}
 				]
 			}
 			CheckBox { name: "meanstructure";	label: qsTr("Include mean structure")							}
-			CheckBox { name: "int.ov.fixed";	label: qsTr("Fix manifest intercepts to zero")					}
-			CheckBox { name: "int.lv.fixed";	label: qsTr("Fix latent intercepts to zero");	checked: true	}
+			CheckBox { name: "manifestInterceptZeroFix";	label: qsTr("Fix manifest intercepts to zero")					}
```
[the remark this answers is not in the document]
Comment (Kucharssim) [reply inst/qml/SEM.qml:157]: the same for `latentIntercept`

File: inst/qml/MediationAnalysis.qml [hunk 2]
```diff
@@ -44,7 +44,7 @@ Form
         AssignedVariablesList
         {
 			title: qsTr("Outcome")
-            name:  "dependent"
+            name:  "dependents"
```
Comment (Kucharssim) [inline inst/qml/MediationAnalysis.qml:47]:
```suggestion
            name:  "outcomes"
```

File: inst/qml/MediationAnalysis.qml [hunk 3]
```diff
@@ -61,46 +61,46 @@ Form
...
         {
             CIField {
                 text: qsTr("Confidence intervals")
-                name: "ciWidth"
+                name: "ciLevel"
             }
             RadioButtonGroup {
                 title: qsTr("Method")
-                name: "se"
+                name: "samplingMethod"
```
Comment (Kucharssim) [inline inst/qml/MediationAnalysis.qml:84]:
```suggestion
                name: "errorCalculationMethod"
```

File: inst/Upgrades.qml [hunk 4]
```diff
@@ -58,4 +58,858 @@ Upgrades
 			}
 		}
 	}
+
+	Upgrade
+	{
+		functionName: "LatentGrowthCurve";	fromVersion: "0.15";	toVersion: "0.16";
```
Comment (Kucharssim) [inline inst/Upgrades.qml:64]: can you change the version in `DESCRIPTION` and `inst/Description.qml` to `0.16.4`? Then use the `fromVersion: 0.16.3` and `toVersion: 0.16.4`. Also, perhaps put those on separate lines for better readability :)

File: inst/Upgrades.qml [hunk 5]
```diff
@@ -58,4 +58,858 @@ Upgrades
...
+
+		ChangeJS
+		{
+			name:		"emulation"
+			jsFunction:	function(options)
+			{
+				switch(options["emulation"])
+				{
+					case "Mplus":	return "mplus";
+					case "EQS":	return "eqs";
```
Comment (Kucharssim) [inline inst/Upgrades.qml:164]: if there are RadioButtons that were not changed like here (i.e., `lavaan`), you'll need to catch them with a default statement
```suggestion
					case "EQS":	return "eqs";
					default:	return options["emulation"];
```

**Review by Kucharssim — CHANGES_REQUESTED** [review]
Looks good so far, suggestions below

also, please rename `modelName` -> `name` (in the `models` `TabView` in SEM)

**Review by Kucharssim — APPROVED** [review]

**Pull request merged**